src/services/WebAzureSubscriptionProvider.ts

```diff
@@ -103,9 +109,10 @@ class VSCodeAzureSubscriptionProvider extends vscode.Disposable implements Azure
 
     get filters(): AzureSubscription[] {
         const selectedSubscriptionIds = settingUtils.getGlobalSetting<string[] | undefined>('selectedSubscriptions');
-        return this.allSubscriptions
-            .filter(s => selectedSubscriptionIds === undefined || selectedSubscriptionIds.includes(s.subscriptionId))
+        const subscriptions = this.allSubscriptions
+            .filter(s => selectedSubscriptionIds === undefined || selectedSubscriptionIds.includes(`${s.tenantId}/${s.subscriptionId}`))
```

Comment:
I think we should treat `[]` as equivalent to `undefined`, i.e., nothing hidden by filter:
```suggestion
            .filter(s => selectedSubscriptionIds === undefined || selectedSubscriptionIds === [] || selectedSubscriptionIds.includes(`${s.tenantId}/${s.subscriptionId}`))
```